railties/test/generators/shared_generator_tests.rb

```diff
@@ -154,13 +154,6 @@ def test_template_is_executed_when_supplied_an_https_path
     assert_equal url, applied
   end
 
-  def test_skip_git
-    run_generator [destination_root, "--skip-git", "--full"]
-    assert_no_file(".gitignore")
-    assert_no_file(".gitattributes")
-    assert_no_directory(".git")
-  end
```

Comment:
This test is here so that it can also be run by [`PluginGeneratorTest`](https://github.com/rails/rails/blob/8c23eaa4eb328e4d007507cee0cc4a46a3e803f0/railties/test/generators/plugin_generator_test.rb#L25-L36).  By moving it to `AppGeneratorTest`, you reduce test coverage.